src/ServiceInsight/MessageViewers/XmlViewer/XmlMessageViewModel.cs

```diff
@@ -55,5 +60,24 @@ public void Clear()
         {
             messageView?.Clear();
         }
+
+        private string GetFormatted(string message)
+        {
+            try
+            {
+                using (var stringReader = new StringReader(message))
+                using (var xmlReader = new XmlTextReader(stringReader))
+                {
+                    var xml = XDocument.Load(xmlReader);
+                    return xml.ToString();
+                }
+            }
+            catch (Exception)
```

Comment:
Aha! This is where the problem is fixed. Previously we were catching `XmlException`.